### DISCUSSION TITLE
V2: Menu "Extended Settings" when creating new timesheet

### DISCUSSION THREAD

**Discussion by HeinzWuert** [kimai/kimai#3753] [Questions & Answers]
Is ist possible  to blank out this menu point via settings?
For a better workflow I would like to see instead the input field for expenses.

![image](https://user-images.githubusercontent.com/45666972/212133796-873dc3cc-e16c-411f-a2c4-64215ec396b2.png)

**Comment by kevinpapst** [comment]
No.

You can repeat the expense form topic 100000 times 😁 but it will not make it more likely. 
I said before and I can repeat: it is way too complicated for a feature just one customer wants.

**Comment by HeinzWuert** [comment]
constant dripping wears away the stone  :-)
